src/plugin.py

```diff
@@ -358,11 +359,26 @@ def _get_not_added_free_games(owned_games):
             self.local_games_called = True
 
     async def get_game_time(self, game_id, context):
-        game_time_minutes = None
+        total_time = None
+        last_played_time = None
+
         if game_id == "5272175":
-            game_time_minutes = await self._get_overwatch_time()
-            log.debug(f"Gametime for Overwatch is {game_time_minutes} minutes.")
-        return GameTime(game_id, game_time_minutes, None)
+            total_time = await self._get_overwatch_time()
+            log.debug(f"Gametime for Overwatch is {total_time} minutes.")
+
+        for game in self.local_client.config_parser.games:
+            try:
+                blizzard_game = Blizzard[game.uid]
+            except KeyError:
+                log.warning(f'[{game.uid}] is not known blizzard game. Skipping')
+                break
```

Comment:
actually in the `Blizzard`  has a searching process, so here break is safe.

Reply:
@bartok765 there is no need to create another new test file , so i add another test function,  the None and the had a data situation all have been tested.

also i need to change the local_client_mock to been a `MagicMock` to make sure it iterable, for the sake of that piece of code: `for game in self.local_client.config_parser.games`

Reply:
@bartok765 first ,we are sure about that the galaxy client call this function with one game id, then use the Blizzard class found the target blizzard_game class. 

second if here program can not found ,  that mean this plugin can not install this game at all , because this plugin has many place is based on the `Blizzard` class load function. so in that siuation which you said the PTR version game, we should add some game definition.py , but not here problem.